src/components/member-profile/contribution-type/index.js

```diff
@@ -115,23 +85,16 @@ const ContributionType = (props) => {
       <div className={showMoreContentClass}>
         {type !== 'Active tasks' ? (
           <div>
-            {devUser
-              ? renderContributions(
-                  contributions,
-                  fullName,
-                  imageLink,
-                  devUser,
-                  type
-                )
-              : renderContributions(
-                  contributions,
-                  fullName,
-                  imageLink,
-                  devUser
-                )}
+            {renderContributions(
+              contributions,
+              fullName,
+              imageLink,
+              devUser,
```

Comment:
Why used devUser prop??

Reply:
It was already there before, as some old code still use featureflag in their flow. So we cant remove it as discussed in other comments. Thanks Shubham